### ISSUE TITLE
lfm recalculate function seems return wrong vertex

### ISSUE THREAD

**Issue by icecity96** [GiulioRossetti/cdlib#97]
```python
    def recalculate(self):
        for vid in self.nodes:
            fitness = self.cal_remove_fitness(vid)
            if fitness < 0.0:
                return vid
        return None
```
If return the `vid` will cause the `new_fitness` lower than `old_fitness`.
The right condition should be:
```
if fitness > 0.0:
```

**Comment by Yquetzal** [comment]
Dear @icecity96 , thanks for you contribution!
I'm not an expert of this method, but I tried to check.
1)If I try the change you propose, the function seems to fail.
2)So I try to understand the logic:
-The highest the fitness, the best the score. 
-a score >0 means that the NEW fitness obtained is better than the OLD fitness. 
-So a score <0 means that the OLD fitness was better (before the changes we just made ?)
-So if the score is <0, maybe we should go back to the OLD situation, therefore the situation in which the current node was not in the community
=> So if the score is <0, we remove the node.

I'm not certain of my reasoning, because I'm not expert of this method. Feel free to correct my arguments !
